holoviews/plotting/bokeh/widgets.py

```diff
@@ -27,14 +27,8 @@ class BokehServerWidgets(param.Parameterized):
     and dropdown widgets letting you select non-numeric values.
     """
 
-    basejs = param.String(default=None, precedence=-1, doc="""
-        Defines the local CSS file to be loaded for this widget.""")
-
-    extensionjs = param.String(default=None, precedence=-1, doc="""
-        Optional javascript extension file for a particular backend.""")
-
-    css = param.String(default=None, precedence=-1, doc="""
-        Defines the local CSS file to be loaded for this widget.""")
+    editable = param.Boolean(default=False, doc="""
+        Whether the slider text fields should be editable.""")
```

Comment:
Should mention the  the reason it is disabled by default. It sounds useful, so there is a reason we want it off by default...